test/FastExpressionCompiler.TestsRunner/Program.cs

```diff
@@ -226,6 +226,9 @@ void Run(Func<int> run, string name = null)
                 Run(new Issue308_Wrong_delegate_type_returned_with_closure().Run);
                 Run(new FastExpressionCompiler.LightExpression.IssueTests.Issue308_Wrong_delegate_type_returned_with_closure().Run);
 
+                Run(new Issue310_InvalidProgramException_ignored_nullable().Run);
+                Run(new FastExpressionCompiler.LightExpression.IssueTests.Issue310_InvalidProgramException_ignored_nullable().Run);
+
```

Comment:
The same test run should be added to net472 test runner, another Program.cs
Otherwise, great PR. Thank you. Waiting for this change and merging.

Reply:
Ok good point, should be fixed now